CMakeLists.txt

```diff
@@ -2219,29 +2217,49 @@ if(FAAD)
   endif()
 endif()
 
+# FDK-AAC is loaded dynamically at runtime by EncoderFdkAac using QLibrary,
+# so copy it into the Windows and macOS packages, but do not link to it.
+find_library(FDK_AAC_LIBRARY fdk-aac)
+# On Windows find_library finds the .lib file, but the installer needs the .dll file.
+find_file(FDK_AAC_DLL fdk-aac.dll PATH_SUFFIXES ${CMAKE_INSTALL_BINDIR})
+if(APPLE AND MACOS_BUNDLE AND FDK_AAC_LIBRARY)
+  message(STATUS "Found fdk-aac: ${FDK_AAC_LIBRARY}")
+  file(COPY ${FDK_AAC_LIBRARY}
+    DESTINATION ${MIXXX_INSTALL_PREFIX}/Contents/Frameworks
+    FOLLOW_SYMLINK_CHAIN
+  )
+elseif(WIN32 AND FDK_AAC_DLL)
+  message(STATUS "Found fdk-aac DLL: ${FDK_AAC_DLL}")
+  install(FILES ${FDK_AAC_DLL} DESTINATION ${MIXXX_INSTALL_BINDIR})
+elseif(FDK_AAC_LIBRARY)
+  message(STATUS "Found fdk-aac: ${FDK_AAC_LIBRARY}")
```

Comment:
This message is pointless.

Reply:
What? No it isn't. It has been really helpful debugging this on CI.